webmails/roundcube/Dockerfile

```diff
@@ -21,6 +21,7 @@ ENV ROUNDCUBE_URL https://github.com/roundcube/roundcubemail/releases/download/1
 RUN apt-get update && apt-get install -y \
       zlib1g-dev libzip4 libzip-dev \
       python3-jinja2 \
+      gpg
```

Comment:
I would say this will break the build because of a missing backslash.

Reply:
Whoopsies. Fixed.